ez-content-view.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="ez-server-side-content.html">
+
+<style>
+ez-content-view {
+    display: flex;
+}
+
+ez-content-view .ez-content-view-column {
```

Comment:
this is not the correct place for this CSS, this should be in hybrid-platform-ui because this package is responsible for generating the elements you are styling here.